src/utils/buffer_utils.cpp

```diff
@@ -26,7 +26,6 @@ template <typename T> void PackData(const AthenaArray<T> &src, T *buf,
   for (int n=sn; n<=en; ++n) {
     for (int k=sk; k<=ek; k++) {
       for (int j=sj; j<=ej; j++) {
-#pragma omp simd
```

Comment:
did these pragmas cause issues?

Reply:
> did these pragmas cause issues?

I removed these lines because of the issue reported here:

https://github.com/PrincetonUniversity/athena/issues/475

Is this taken care of in the master version?